packages/connectkit/src/index.ts

```diff
@@ -5,7 +5,12 @@ export { useModal } from './hooks/useModal';
 export { SIWEProvider, useSIWE } from './siwe';
 export type { SIWESession, SIWEConfig } from './siwe';
 
-export { ConnectKitProvider, Context } from './components/ConnectKit';
+export {
+  ConnectKitProvider,
+  Context,
+  useContext,
+  routes,
```

Comment:
@0xkwad with regard to exporting the `routes` and `useContext` for navigating the modal, we have options for navigating the modal by using the [`useModal` Hook](https://docs.family.co/connectkit/api-reference#hooks-usemodal) that does some checks before navigating since there's a few [routes that aren't navigable in certain cases](https://github.com/family/connectkit/blob/348090af8698031fa0145020d4e8cf9c5d0119b2/packages/connectkit/src/hooks/useModal.ts#L51-L80) without causing errors.

If there is a use-case where you need these that the `useModal` Hook can't accomplish I don't mind exporting these, but we won't write documentation about this since it would be very advance + experimental usage that is subject to change.

Reply:
Oh neat, didn't realize it was added 😅
I remember it only having `open` and `setOpen`

That's pretty much the only use case we have for these, so fine to not export 👍